core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1019,3 +1025,11 @@
     - cluster: app.ci
       namespace: prow-logging
       name: vector-agent-secret
+- from:
+    bw_password:
+      bw_item: "OpenShift CI Bitwarden Bot"
+      attribute: password
+  to:
+    - cluster: app.ci
+      namespace: ci
+      name: bitwarden-credentials-OpenShift-CI-Bitwarden-Bot
```

Comment:
I believe uppercase letters are not allowed for object names

Reply:
Done.